src/MSVC/vs.h

```diff
@@ -0,0 +1 @@
+#define not !
```

Comment:
Adding two separate source directories for these several lines seems too much. Could you just merge this file into common.hh?